global.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sdk": {
+    "version": "8.0.100",
+    "allowPrerelease": false,
+    "rollForward": "latestFeature"
+  }
+}
```

Comment:
Added to ensure that local runs use same SDK as CI runs.

Reply:
:+1: but I'd to this in a separate PR, as this change isn't related to what the PR title says.

Reply:
Added:
- #591